tamr_unify_client/models/attribute/collection.py

```diff
@@ -0,0 +1,94 @@
+from tamr_unify_client.models.attribute.resource import Attribute
+from tamr_unify_client.models.base_collection import BaseCollection
+
+
+class AttributeCollection(BaseCollection):
+    """Collection of :class:`~tamr_unify_client.models.attribute.resource.Attribute` s.
+
+    :param client: Client for API call delegation.
+    :type client: :class:`~tamr_unify_client.Client`
+    :param data: JSON data representing this resource
+    :type data: dict
+    :param api_path: API path used to access this collection.
+        E.g. ``"datasets/1/attributes"``.
```

Comment:
I was going to comment that a comma should follow E.g., but according to [this](https://www.dailywritingtips.com/comma-after-i-e-and-e-g/) article, it is not an agreed upon rule. So LGTM.